app/code/Magento/GoogleAnalytics/Block/Ga.php

```diff
@@ -155,6 +156,16 @@ public function getOrdersTrackingCode()
         return implode("\n", $result);
     }
 
+    /**
+     * Google Optimize container ID, Example: GTM-XXXXXX
+     *
+     * @return string
+     */
+    public function getOptimizeContainerId()
```

Comment:
@ihor-sviziev you made it breaking compatibility since the class is marked with `@api`

Reply:
Consider please doing it in a compatible way or just add a new block for this functionality